gf-ui-lib/src/components/Button/Button.tsx

```diff
@@ -16,19 +17,29 @@ interface ButtonProps {
    * Button type
    */
   type: "submit" | "button";
+  /**
+   * Button size.
+   * If size is undefined, then the button will have parentWidth style
+   */
+  size?: "innerContent" | "parentWidth";
 }
 
 /**
  * Button component
  */
 export const Button: React.FC<ButtonProps> = (props: ButtonProps) => {
+
+  const buttonSize = props.size ? props.size : "";
```

Comment:
Nope) unscalable solution) and size can't be null or underfind due to default props

Reply:
Why don't u want to throw prop into return without additional constant?